lib/shanty_docker_plugin/output_parser.rb

```diff
@@ -0,0 +1,48 @@
+require 'shanty_docker_plugin/error'
+require 'shanty/logger'
+
+module DockerPlugin
+  # Methods for parsing the output from docker
+  module OutputParser
```

Comment:
Tempted to say this should be called `DockerOutputParser` - it wasn't clear to me what this was until I noticed the comment above.